modules/robin/src/test/ts/atomic/api/TextZoneTest.ts

```diff
@@ -1,14 +1,15 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Assert, context, describe, it } from '@ephox/bedrock-client';
 import { Gene, TestUniverse, TextGene } from '@ephox/boss';
 import { Fun, Optional } from '@ephox/katamari';
+import { assert } from 'chai';
```

Comment:
The rest of the test uses bedrock `Assert`, do we really need both in one file?

Reply:
I guess we could use the Bedrock assert. But we have this odd code base where some places uses  bedrock Assert and some users chai and the bedrock version is inferior and has its parameters in a completely different order.
I think at some point we should normalize to chai or expose everything chai supports on bedrock if we are to continue using that. I guess we could write  code mod or run an AI to migrate all of them.